command/peers_add.go

```diff
@@ -13,31 +17,34 @@ type PeersAdd struct {
 func (p *PeersAdd) DefineFlags() {
 	if p.flagMap == nil {
 		// Flag map not initialized
-		p.flagMap = make(map[string]FlagDescriptor)
+		p.flagMap = make(map[string]types.FlagDescriptor)
 	}
 
-	p.flagMap["libp2p-address"] = FlagDescriptor{
+	p.flagMap["addr"] = MetaFlagDescriptor{
```

Comment:
The reasoning behind adding the ability to add more peers is just better ux - not having to type out the same command a lot of times. 

How it would work after this PR, if you wanted to add multiple peers:
`go run main.go peers-add -addr <ADDR1> -addr <ADDR2> -addr <ADDR3>` and so on.